docs/articles/nunit/writing-tests/attributes.md

```diff
@@ -21,7 +21,7 @@ This table lists all the attributes supported by NUnit.
 | [NonTestAssembly Attribute](attributes/nontestassembly.md)     | Specifies that the assembly references the NUnit framework, but that it does not contain tests. |
 | [OneTimeSetUp Attribute](attributes/onetimesetup.md)        | Identifies methods to be called once prior to any child tests. |
 | [OneTimeTearDown Attribute](attributes/onetimeteardown.md)     | Identifies methods to be called once after all child tests. |
-| [Order Attribute](attributes/order.md)               | Specifies the order in which decorated test should be run (against others). |
+| [Order Attribute](attributes/order.md)               | Specifies the order in which decorated test should be run within the contining fixture or suite. |
```

Comment:
s/contining/containing/

Reply:
I'd love to see a PR or issue for this if you're willing. :) Otherwise I'll capture it as part of https://github.com/nunit/docs/issues/362